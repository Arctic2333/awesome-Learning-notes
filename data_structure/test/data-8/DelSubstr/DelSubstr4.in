888666aaaaaaaaaaaaaaaaadefaaaadefdefaaaadefdefdef999000aaaaaaaaaaaaadefaaaadefdefaaaadefdefdef&&aaaaaaaaaaaaaadefaaaadefdefaaaadefdefdef
aaadef