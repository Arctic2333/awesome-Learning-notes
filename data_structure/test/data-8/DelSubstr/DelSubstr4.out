888666aaaaaaa999000aaa&&aaaa
